library/globals.inc.php

```diff
@@ -3092,5 +3092,40 @@ function gblTimeZones()
             xl('Select to print patient address labels from popups in left nav screen.')
         ),
 
+        'env_x_width' => array(
+            xl('Envelope Width in mm'),
+            'num',                           // data type
+            '104.775',                              // default = false
+            xl('In Portrait mode, determines the width of the envelope along the x-axis in mm')
+        ),
+
+        'env_y_height' => array(
+            xl('Envelope Height in mm'),
+            'num',                           // data type
+            '241.3',                              // default = false
+            xl('In Portrait mode, determines the height of the envelope along the y-axis in mm')
+        ),
+
+        'env_font_size' => array(
+            xl('Font Size in Pt'),
+            'num',                           // data type
+            '14',                              // default = false
+            xl('Sets the font of the address text on the envelope in mm')
+        ),
+
+        'env_x_dist' => array(
+            xl('Envelope x-axis starting pt'),
+            'num',                           // data type
+            '65',                              // default = false
+            xl('Distance from the \'top\' of the envelope in mm')
+        ),
+
+        'env_y_dist' => array(
+            xl('Envelope y-axis starting pt'),
+            'num',                           // data type
+            '220',                              // default = false
+            xl(' Distance from the right most edge of the envelope in portrait position in mm')
+        ),
+
```

Comment:
This is a bit of a tough one, because allowing users to set this means every user will need to set it (rather than just once when new paper stock or printer), which would also be messy. An option here would be to have an option to assign an aco to a global, thus allowing more fine grained modification of globals(so could then assign group of users ability to assign some globals and not others). This approach is currently being done for CDR rules and forms. Obviously, this stuff is outside the scope of growlingflea's work.